Lib/test/test_long.py

```diff
@@ -204,7 +204,10 @@ def test_division(self):
         self.check_division(710031681576388032, 26769404391308)
         self.check_division(1933622614268221, 30212853348836)
 
-
+    @support.cpython_only
+    def test_division_with_one(self):
```

Comment:
If you test assertIs(), please add a comment mentioning bpo-41902 and explain that it's a micro-optimization which returns x for x//1 if x is an int made of a single digit.